content/working_with/official_plugins/Utilities/rollback.md

```diff
@@ -0,0 +1,336 @@
+---
+layout: bt_wiki
+title: Rollback Plugin
+category: Official Plugins
+draft: false
+weight: 100
+---
+{{% note %}}
+These features are part of the **utilities plugin**.
+{{% /note %}}
+
+# Cloudify Utilities: Rollback Workflow
+
+## Description
+Add support for rollback node instances that exists in unresolved states due to failure in install workflow.
+Also, wrappers workflows for {{< param product_name >}} lifecycle operations introduced.
+
+## Prerequisites
+* Tested with Cloudify 5.1.1.
+
```

Comment:
Add deprecation notice for 5.2, that it will be moved to Cloudify Common and will be a built-in workflow.

Reply:
Done.